devtools/server/actors/replay/connection.js

```diff
@@ -533,6 +527,114 @@ class Recording extends EventEmitter {
   }
 }
 
+function checkShouldValidateUrl() {
+  if (gShouldValidateUrl !== null) {
+    return Promise.resolve(gShouldValidateUrl)
+  } else {
+    return queryAPIServer(`
+      query GetOrgs {
+        viewer {
+          workspaces {
+            edges {
+              node {
+                isOrganization
+                settings {
+                  features
+                }
+              }
+            }
+          }
+        }
+      }
+    `).then(resp => {
+      if (resp.errors) {
+        throw new Error("Unexpected error checking Replay user permissions");
+      }
+
+      const workspaces = resp.data.viewer.workspaces.edges;
```

Comment:
Good point. Forgot about workspace API keys. Suggest we ignore this for now because we can handle it more cleanly when this feature is managed by the protocol.

Reply:
Yeah no problem with not handling them for now, but we definitely shouldn't throw a JS error :)

Reply:
It'll be caught by `canRecordUrl()` and logged in the console and sent to honeycomb. I doubt we'll see it in the short time this will be live.

Reply:
I'm a bit confused, your code is already written to handle this with `!workspaces ? false : ` so why not add the `?` to do `resp.data.viewer?.workspaces.edges;`? That's the only way for `workspaces` to be falsy to begin with.

Reply:
:+1: